### DISCUSSION TITLE
Strongswan fails to start and initiate VPN connection from network manager

### DISCUSSION THREAD

**Discussion by thystra** [strongswan/strongswan#1441] [Q&A]
**System (please complete the following information):**
 - OS: Ubuntu 22.10 Jammy
 - Kernel version (if applicable): 5.19.0-23-generic
 - strongSwan version(s): 6.0 Beta1 and Network Manager 1.6.0
 - Tested/confirmed with the latest version: yes

**Describe the bug**
Attempt to configure and initiate IKEv2 VPN via Network Manager in Ubuntu Desktop.  VPN fails to start due to program unable to find charon-nm program. 

**To Reproduce**
1. Enable VPN connection in Network manager

**Expected behavior**

**Logs/Backtraces**
```
Nov 15 14:50:00 gryffon NetworkManager[1317]: <info>  [1668541800.9088] vpn[0x56515c1ba350,a1a84b2d-eee0-41fb-b3ed-aab07499ee50,"Greki IKE2"]: starting strongswan
Nov 15 14:50:00 gryffon NetworkManager[1317]: <info>  [1668541800.9092] audit: op="connection-activate" uuid="a1a84b2d-eee0-41fb-b3ed-aab07499ee50" name="Greki IKE2" pid=21786 uid=1000 result="success"
Nov 15 14:50:00 gryffon NetworkManager[1317]: <warn>  [1668541800.9123] vpn[0x56515c1ba350,a1a84b2d-eee0-41fb-b3ed-aab07499ee50,"Greki IKE2"]: starting: failure to start VPN service: Failed to execute child process “/usr/local/libexec/ipsec/charon-nm” (No such file or directory)
```

```alan@gryffon:~/Downloads$ find charon-nm
find: ‘charon-nm’: No such file or directory
```

**Additional context**
I started out with the distro packages.  They then were unable to load the configuration editor.  I compiled and installed strongswan plugin 1.6 and 6.0Beta 1.  Now, syslog is showing that charon-nm is not found.  Charon-nm, if required, should be installed as part of the compile and install process.

**Comment by Thermi** [comment]
It's because you didn't build charon-nm. You need to enable building it.
